### PR TITLE
[bitnami/prestashop] Revisit tests

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

After reviewing the existing tests and the general testing strategy we have been following until now, we saw there was a need to revisit them. The main objective that our chart testing strategy should aim for is asserting that the Chart deliverable works. As such, the focus should not be on the application itself or the container, which are regarded as trustful components (that is, they should have been tested at a previous stage).

As charts usually pack a number of different components, it is essential to test that their integrations and the application itself work as expected.

Consequently, we have updated the tests so they are related to the chart deliverable and left only a couple of basic tests to verify that the application functions correctly. Besides that, an effort has been made to reduce future tests' flakiness improving some existing selectors and checks.

Tested in https://github.com/joancafom/charts/pull/102

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/prestashop/cypress/cypress/integration/prestashop_spec.js [hunk 1]
```diff
@@ -12,119 +12,32 @@ it('allows a user to place an order and an admin to list it', () => {
...
-  const DEFAULT_ACTIVE_PAY_METHODS = ['Bank transfer', 'Payments by check'];
-
-  cy.contains('div.card', 'Active payment').within(() => {
-    DEFAULT_ACTIVE_PAY_METHODS.forEach((method) => {
-      cy.contains(method);
-    });
-  });
-});
-
-it('allows performing backups', () => {
```
Comment (FraPazGal) [inline .vib/prestashop/cypress/cypress/integration/prestashop_spec.js:106]: I suppose this test was added to check prestashop's PV. Are we using the PV in any way with the remaining test or chart installation? If not, I would include again this backup test.

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @joancafom, please check my comments when possible!

File: .vib/prestashop/cypress/cypress/integration/prestashop_spec.js (hunk 1 above)
Comment (joancafom) [reply .vib/prestashop/cypress/cypress/integration/prestashop_spec.js:106]: Agreed, I added it back. Thanks!

**Review by FraPazGal — APPROVED** [review]
LGTM, thanks!

**Pull request merged**